src/CattyTests/DataModel/UserVariableTests.swift

```diff
@@ -70,4 +70,43 @@ final class UserVariableTests: XCTestCase {
         XCTAssertFalse(userVariableA === userVariableCopy)
         XCTAssertTrue(userVariableB === userVariableCopy)
     }
+
+    func testIsEqualToUserVariableForEmptyInit() {
+        let userVariableA = UserVariable()
+        userVariableA.name = "userVar"
+
+        let userVariableB = UserVariable()
+        userVariableB.name = "userVar"
+
+        userVariableA.value = NSMutableArray(array: [50, 51])
+        userVariableB.value = NSMutableArray(array: [50, 52])
+        XCTAssertFalse(userVariableA.isEqual(to: userVariableB))
+
+        userVariableB.value = NSMutableArray(array: [50, 51])
+        XCTAssertTrue(userVariableA.isEqual(to: userVariableB))
+    }
+
+    func testIsEqualToUserVariableForVariable() {
+        let userVariableA = UserVariable(name: "userVar", isList: false)
+        let userVariableB = UserVariable(name: "userVar", isList: false)
+
+        userVariableA?.value = NSMutableArray(array: [50, 51])
```

Comment:
Please assign a string since only lists can have arrays (and do the same in another tests for the second other value variables can have: strings).